### PR TITLE
Fix missing amcl and gmapping launch files in turtlebot_navigation while creating/navigating  map

### PR DESCRIPTION
**Pull request by mzaiady**
Fix error No such file or directory gmapping.launch.xml and amcl.launch.xml under turtlebot_navigation/launch/includes/

while calling
`roslaunch turtlebot_gazebo amcl_demo.launch`
and also while calling 
`roslaunch turtlebot_gazebo gmapping_demo.launch`

And this because of changing the  gmapping.launch.xml and amcl.launch.xml  on turtlebot_navigation

Created issue: 
https://github.com/turtlebot/turtlebot_apps/issues/159

### CODE REVIEW

**Comment by mzaiady** [comment]
By the way , this my first pull request on GitHup, so I hope I did not forget something!

File: turtlebot_gazebo/launch/amcl_demo.launch [hunk 1]
```diff
@@ -1,18 +1,22 @@
-<launch>
+launch>
```
Comment (mzaiady) [inline turtlebot_gazebo/launch/amcl_demo.launch]: it looks like, removed the opening by mistake, i will provide fix today

File: turtlebot_gazebo/launch/amcl_demo.launch [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+  <arg name="3d_sensor" default="$(env TURTLEBOT_3D_SENSOR)"/>  <!-- r200, kinect, asus_xtion_pro -->
+
   <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
 
   <!-- Localization -->
   <arg name="initial_pose_x" default="0.0"/>
   <arg name="initial_pose_y" default="0.0"/>
   <arg name="initial_pose_a" default="0.0"/>
-  <include file="$(find turtlebot_navigation)/launch/includes/amcl.launch.xml">
+ <arg name="custom_amcl_launch_file" default="$(find turtlebot_navigation)/launch/includes/amcl/$(arg 3d_sensor)_amcl.launch.xml"/>
```
Comment (stonier) [inline turtlebot_gazebo/launch/amcl_demo.launch]: Nitpicking sorry, extra space needed here.

**Comment by stonier** [comment]
LGTM - @kevincwells did you test these?

**Comment by mzaiady** [comment]
I tested modification on my existing/install version, and it works fine

Then I did the same changes to the branch version, but I did not install/test the copied changes
of-course, I it must be test, but I'm new to ROS  and only used catkin_make tool, never used rosinstall.

I will try find some time and install it using rosinstall and see.

**Comment by rohbotics** [comment]
@stonier 
This patch works in my testing, for both gmapping and amcl.

I think it is good to merge once the white-space is fixed.

**Pull request merged**